helper/chunkify_test.js

```diff
@@ -0,0 +1,40 @@
+
+function chunkify(message, maxChars){
+    let lines = message.split(/\n/);
+    let chunks = [];
+    let numChars = 0;
+    for (let i = 0; i < lines.length; i++){
+        numChars = lines[i].length;

```

Comment:
if numchars is not used outside of this scope, then it should be declared const here

Reply:
numChars was update in the different loops